plugin/pkg/admission/limitranger/admission_test.go

```diff
@@ -30,14 +30,17 @@ import (
 	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/fake"
 )
 
-func getResourceList(cpu, memory string) api.ResourceList {
+func getResourceList(cpu, memory, storage string) api.ResourceList {
```

Comment:
this is a style thing, but since we will not call with all 3 values populated, can we have something like:

```
getComputeResourceList(cpu,memory string)
getStorageResourceList(storage string)
```

It will help us keep things sensible if storage ever does more, and wont confuse compute resource list.